get_all_manifests.sh

```diff
@@ -10,13 +10,13 @@ declare -A COMPONENT_MANIFESTS=(
     ["ray"]="opendatahub-io:kuberay:master:ray-operator/config:ray"
     ["kueue"]="opendatahub-io:kueue:dev:config:kueue"
     ["data-science-pipelines-operator"]="opendatahub-io:data-science-pipelines-operator:main:config:data-science-pipelines-operator"
-    ["odh-dashboard"]="opendatahub-io:odh-dashboard:incubation:manifests:dashboard"
+    ["odh-dashboard"]="opendatahub-io:odh-dashboard:main:manifests:dashboard"
     ["kf-notebook-controller"]="opendatahub-io:kubeflow:v1.7-branch:components/notebook-controller/config:odh-notebook-controller/kf-notebook-controller"
     ["odh-notebook-controller"]="opendatahub-io:kubeflow:v1.7-branch:components/odh-notebook-controller/config:odh-notebook-controller/odh-notebook-controller"
     ["notebooks"]="opendatahub-io:notebooks:main:manifests:notebooks"
-    ["trustyai"]="trustyai-explainability:trustyai-service-operator:release/1.10.2:config:trustyai-service-operator"
-    ["model-mesh"]="opendatahub-io:modelmesh-serving:release-0.11.1:config:model-mesh"
-    ["odh-model-controller"]="opendatahub-io:odh-model-controller:release-0.11.1:config:odh-model-controller"
+    ["trustyai"]="trustyai-explainability:trustyai-service-operator:main:config:trustyai-service-operator"
+    ["model-mesh"]="opendatahub-io:modelmesh-serving:main:config:model-mesh"
```

Comment:
If I'm not wrong, for model-mesh this is still expected to be `release-0.11.1`.
If the nightly is using `main`, I think we should fix the nightly.

Reply:
leave this to you and your team to decide if keep "release-0.11.1"
- either we fix in midstream PR, (switch to release-0.11.1)
- or apply this PR change here

but i think you and team need to consider to have a dedicate branch for nightly and tags for odh releases in general (as other teams are doing)

Reply:
We do incubate code, but the incubation branch is typically promoted from incubation quickly.
I'll confirm on Monday.

Reply:
https://github.com/opendatahub-io/modelmesh-serving/branches
these are the availbe branches: main, release-0.11.1, 
i cannot find "incubating" @Jooho which you want to use, this is not for release, this is for odh-nightly build

Reply:
Our incubating release is, for now, the `release-0.11.1` branch :-)
So, if I understood Jooho, we do want to keep here model-mesh unchanged and fix odh-nighly.

Reply:
changes reverted